android/iden3coreapi/src/main/java/com/iden3/iden3coreapi/Iden3IdentityFactory.kt

```diff
@@ -0,0 +1,125 @@
+package com.iden3.iden3coreapi
+
+import iden3mobile.*
+import java.io.File
+import java.io.FileNotFoundException
+import java.lang.IllegalArgumentException
+
+open class Iden3IdentityFactory {
+
+    companion object {
+        @JvmStatic lateinit var instance: Iden3IdentityFactory
+    }
+
+    init {
+        instance = this
+    }
+
+    private lateinit var web3Url: String
+    private lateinit var storePath: String
+    private var checkTicketsPeriod : Long = 10000
+
+    /**
+     * Initialize the Iden3IdentityFactory library with the parameters need to setup.
+     *
+     * @param web3Url: String the Web3 url.
+     * @param storePath: String the absolute path where to store the identities.
+     * @param checkTicketsPeriod: Time in miliseconds of the period needed for checking the tickets.
+     *
+     * @return Boolean if the initialization has been successful.
+     */
+    fun initializeAPI(web3Url: String, storePath: String, checkTicketsPeriod: Long) : Boolean {
```

Comment:
I wouldn't add the API part (just `initialize`)

Reply:
done